double '<' character in csproj files

Hello!

I started learning dotnet core few days ago and found this package.

I started by creating a dotnetcore webapplication in 1.1 framwork and came across this error message.

> Testa.csproj(30,6): error MSB4025: The project file could not be loaded. Name cannot begin with the '<' character, hexadecimal value 0x3C. Line 30, position 6.

When I manually remove the double '<' the "restore" and "build" proceed normally...

I think the right solution is removing the doubled '<' from CompanyWebApplication1.* files

changing this:

`<<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />`

to:
`<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />`

--- a/templates/projects/fsharp_mvcbasic/Company.WebApplication1.fsproj
+++ b/templates/projects/fsharp_mvcbasic/Company.WebApplication1.fsproj
@@ -31,7 +31,7 @@
     <PackageReference Include="Microsoft.AspNetCore" Version="1.1.1" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.2" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="1.1.1" />
-    <<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
     <PackageReference Include="Microsoft.FSharp.Core.netcore" Version="1.0.0-alpha-161023" />
     <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="1.1.0" />
   </ItemGroup>

--- a/templates/projects/fsharp_web/Company.WebApplication1.fsproj
+++ b/templates/projects/fsharp_web/Company.WebApplication1.fsproj
@@ -25,7 +25,7 @@
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0-beta1" /><% } %>
     <PackageReference Include="FSharp.NET.Sdk" Version="1.0.0-beta-*" PrivateAssets="All" />
     <PackageReference Include="Microsoft.AspNetCore" Version="1.1.1" />
-    <<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
     <PackageReference Include="Microsoft.FSharp.Core.netcore" Version="1.0.0-alpha-161023" />
   </ItemGroup>
   <% } %>

--- a/templates/projects/fsharp_webapi/Company.WebApplication1.fsproj
+++ b/templates/projects/fsharp_webapi/Company.WebApplication1.fsproj
@@ -34,7 +34,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="1.1.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="1.1.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration.CommandLine" Version="1.1.0" />
-    <<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
     <PackageReference Include="Microsoft.FSharp.Core.netcore" Version="1.0.0-alpha-161023" />
   </ItemGroup>
   <% } %>

--- a/templates/projects/mvc/Company.WebApplication1.csproj
+++ b/templates/projects/mvc/Company.WebApplication1.csproj
@@ -52,7 +52,7 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer.Design" Version="1.1.1" PrivateAssets="All" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="1.1.0" PrivateAssets="All" />
     <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="1.1.1" />
-    <<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
     <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="1.1.0" />
   </ItemGroup>
   <ItemGroup>

--- a/templates/projects/webapi/Company.WebApplication1.csproj
+++ b/templates/projects/webapi/Company.WebApplication1.csproj
@@ -20,7 +20,7 @@
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0" /><% } %>
     <PackageReference Include="Microsoft.AspNetCore" Version="1.1.1" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.2" />
-    <<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
   </ItemGroup>
   <% } %>
 </Project>

